axiom/tests/test_distributed_trace.c

```diff
@@ -1351,6 +1352,18 @@ static void test_create_trace_state_header(void) {
                         nr_strstr(result, expected));
   nr_free(result);
 
+  /*
+   * Test: locale is set to use `,` instead of `.` for decimal values
+   */
+  dt->priority = 0.123456;
+  expected = "777@nr=0-0-1234-9876-123456789-meatball!-0-0.123456-";
+  result
+      = nr_distributed_trace_create_w3c_tracestate_header(dt, span_id, txn_id);
+  tlib_pass_if_not_null("locale should not affect priority formatting", 
+                        nr_strstr(result, expected));
+  nr_free(result);
+  setlocale(LC_NUMERIC, "en_US");
```

Comment:
Shouldn't https://github.com/newrelic/newrelic-php-agent/commit/8668b955262e9463f790ebe58e2c005648636881 remove this also?
```suggestion
```

Reply:
actually I think it may be better to add the first call to setlocale back in so it can at least be tested locally via dockerfile... otherwise this test ought to be removed altogether

Reply:
like this: 164e8738b3665ec2ddaf209a2005d4c4862d56a8